samples/bot-groupchat-messages-withRSC/nodejs/appPackage/README.md

```diff
@@ -0,0 +1,10 @@
+# App Package
+
+Your "app package" is used to publish your app configuration to Teams by manually using the "Upload a custom app" from the app gallery in Teams. 
```

Comment:
Make sure to link your readme with root file.

Reply:
Readme is linked with root readme file